.gitpod.yml

```
@@ -7,7 +7,7 @@ tasks:
       # Upgrade pyenv itself
       pyenv update
 
-      export PY_VERSIONS="3.8 3.9 3.10 3.11 3.12"
+      export PY_VERSIONS="3.8 3.9 3.10 3.11 3.12 3.13"
```

Support for 3.8 dropped in #315 this is a leftover
```suggestion
      export PY_VERSIONS="3.9 3.10 3.11 3.12 3.13"
```